src/EnergyPlus/PackagedThermalStorageCoil.cc

```diff
@@ -4589,7 +4589,7 @@ void CalcTESCoilDischargeOnlyMode(EnergyPlusData &state, int const TESCoilNum, R
             ElecCoolingPower = TotCap * EIR * RuntimeFraction;
         }
         // now see if coil is running dry
-        PartLoadOutAirEnth = EvapInletEnthalpy - (TotCap * PartLoadRatio) / EvapAirMassFlow;
+        PartLoadOutAirEnth = EvapInletEnthalpy - (TotCap * PLR) / EvapAirMassFlow;
```

Comment:
How did you find this?

Reply:
I can't recall if there was a specific warning. I may just have noticed this while reducing scope of variables and const-ing stuff. 🤷